src/Microsoft.AspNet.Diagnostics/StatusCodeFeature.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Diagnostics
+{
+    public class StatusCodeFeature : IStatusCodeFeature
+    {
+        public StatusCodeFeature()
```

Comment:
How is MVC consuming it? What does it have to take dependency on, now that there is no `[AssemblyNeutral]` support